stockbot.properties.template

```diff
@@ -0,0 +1,2 @@
+[main]
+dbFile=<Absolute path to stockbot sqlite3 db>
```

Comment:
What is an absolute  path?

Reply:
An absolute path starts from the root of the file system (e.g. /home/rohan/data/stocket). A relative path is evaluated relative to the current directory.